crates/symbolicator-service/tests/integration/source_errors.rs

```diff
@@ -124,6 +124,10 @@ async fn test_download_errors() {
 async fn test_deny_list() {
     let (symbolication, _cache_dir) = setup_service(|config| {
         config.cache_dir = None;
+        config.caches.downloaded.retry_misses_after = Some(Duration::ZERO);
+        config.caches.derived.retry_misses_after = Some(Duration::ZERO);
+        // FIXME: `object_meta` caches treat download errors as `malformed`
+        config.caches.derived.retry_malformed_after = Some(Duration::ZERO);
```

Comment:
I believe this has been broken ever since https://github.com/getsentry/symbolicator/pull/516/files#diff-2f856962a69fa5bec61ba23e49b676ce0d95db30e21d49f938da24302cb8b3b4R343-R349

`object_meta` are a `Derived` cache, so server errors were only being retried after the "malformed" timeout since basically forever.